sonar-server/time-travel-server.go

```diff
@@ -82,12 +84,56 @@ func (s *timeTravelServer) Start() {
 	log.Println("start timeTravelServer...")
 }
 
+func toLowerMap(m map[string]interface{}) {
+	for key, val := range m {
+		switch v := val.(type) {
+		case map[string]interface{}:
+			if e, ok := m[key].(map[string]interface{}); ok {
+				toLowerMap(e)
+				m[strings.ToLower(key)] = e
+				if strings.ToLower(key) != key {
```

Comment:
this is minor, but you might want to put `m[strings.ToLower(key)] = e` inside the if scope. Similar for the `m[strings.ToLower(key)] = e` below